backend/ops_api/ops/schemas/portfolios.py

```diff
@@ -0,0 +1,6 @@
+from marshmallow import Schema, fields
+
+
+class PortfolioCansRequestSchema(Schema):
+    year = fields.Integer(required=False)
+    budgetFiscalYear = fields.Integer(required=False)
```

Comment:
pls add updates to OpenApi spec
![SCR-20250307-hzyf](https://github.com/user-attachments/assets/96c0ecad-20be-403d-bf92-ec8243bd168a)